scale-tests/README.md

```diff
@@ -51,10 +51,9 @@ Cluster requirements:
 ```bash
 your-machine $ git clone git@github.com:mesosphere/spark-build.git
 your-machine $ cd spark-build
-your-machine $ docker run -it --net=host -v $(pwd):/spark-build -v </path/to/soak/key>:/ssh/key:ro mesosphere/spark-build:latest
-docker-container # cd /spark-build
-docker-container # pip3 install -r tests/requirements.txt
-docker-container # export PYTHONPATH=$(pwd)/testing:$(pwd)/spark-testing
+your-machine $ docker build -t mesosphere/dcos-commons:${LOGNAME}-spark scale-tests/
```

Comment:
What is `LOGNAME`?

Reply:
Logged in username I believe: `$(whoami)`?

Reply:
(At least that's what `echo $LOGNAME` shows for me)

Reply:
Yes, IIRC $LOGNAME is POSIX, while $USER is not.